app/components/support_interface/reference_with_feedback_component.rb

```diff
@@ -156,9 +152,19 @@ def email_address_row
     end
 
     def relationship_row
+      value = tag.p(relationship, class: 'govuk-body')
+
+      if reference.feedback_provided?
+        value += if relationship_correction.present?
+                   tag.p("#{reference.name} said:", class: 'govuk-body') + tag.p(relationship_correction, class: 'govuk-body')
+                 else
+                   tag.p("This was confirmed by #{reference.name}", class: 'govuk-body')
+                 end
+      end
```

Comment:
suggest extracting a method to get this text, something along the lines of
```
      def relationship_text
        value = tag.p(relationship, class: 'govuk-body')
        return value unless reference.feedback_provided?

        if relationship_correction.present?
          value += tag.p("#{reference.name} said:", class: 'govuk-body')
          value += tag.p(relationship_correction, class: 'govuk-body')
        else
          value += tag.p("This was confirmed by #{reference.name}", class: 'govuk-body')
        end

        value
      end
```

Reply:
Good idea. Done in [8a912d1](https://github.com/DFE-Digital/apply-for-teacher-training/pull/7620/commits/8a912d12dcb84fa471e766386210455a9acbcc2f), thanks. 👍